inst/rmarkdown/templates/wellcomeor/skeleton/elsevier-vancouver.csl

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think we need this file as the skeleton is showing example with a URL. 

We should delete it.